__tests__/app.test.js

```diff
@@ -89,6 +89,44 @@ describe('GET:', () => {
 					expect(body.msg).toEqual('Not found');
 				});
 		});
+		describe('/comments', () => {
+			test('status:200 returns a array of objects with the required properties', () => {
+				return request(app)
+					.get('/api/articles/1/comments')
+					.expect(200)
+					.then((comments) => {
+						comments._body.forEach(comment => {
+							expect(comment).toEqual(
+								expect.objectContaining({
+									comment_id: expect.any(Number),
+									votes: expect.any(Number),
+									created_at: expect.any(String),
+									author: expect.any(String),
+									body: expect.any(String)
+							}))
```

Comment:
each comment should have an article_id, include this in your test here to prove you are getting back the right comments for the requested endpoint! article_id : 1